### PR TITLE
added statusCode to NImageFetchError

### PR DESCRIPTION
**Pull request by albertopeam**

### CODE REVIEW

File: Tests/NMAImageFetchTests/NMAImageFetchTests.swift [hunk 1]
```diff
@@ -89,8 +90,8 @@ final class NMAImageFetchTests: XCTestCase {
         waitForExpectations(timeout: 1)
     }
 
-    func testImageNotFound() {
-        let url = URL(string: "https://www.nomasystems.com/404.png")!
+    func testImageNotFound() throws {
+        let url = try XCTUnwrap(URL(string: "https://github.com/nomasystems/nimagefetch-ios/blob/main/404.png"))
```
Comment (albertopeam) [inline Tests/NMAImageFetchTests/NMAImageFetchTests.swift:94]: "https://www.nomasystems.com/404.png" was returning a 403 so I changed it to get a real 404

**Comment by albertopaITX** [comment]
@albertocsouto @josecriane could you take a look?

**Comment by albertopeam** [comment]
@albertocsouto @josecriane could you take a look?

**Comment by albertopeam** [comment]
> @albertocsouto @josecriane could you take a look?

Hi guys, could you take a look? it should be quickly as it is an small PR 
@albertocsouto @josecriane

**Review by albertocsouto — APPROVED** [review]

**Comment by albertocsouto** [comment]
Thanks!

**Comment by albertopeam** [comment]
Thanks @albertocsouto

**Pull request merged**